server/functions/framework/compiled_function.go

```diff
@@ -148,6 +148,9 @@ func (c *CompiledFunction) String() string {
 			sb.WriteString(", ")
 		}
 		if doltgresType, ok := param.Type().(pgtypes.DoltgresType); ok {
+			if doltgresType.BaseID() == pgtypes.DoltgresTypeBaseID_Unknown {
+				doltgresType = pgtypes.Text
+			}
```

Comment:
We shouldn't need to do this. I'm assuming this was done for `QuoteString`, which doesn't work with `unknown` types. Instead, we should update `QuoteString` to work with them.